contracts/infuser/src/msg.rs

```diff
@@ -0,0 +1,63 @@
+use crate::state::*;
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::Addr;
+
+#[cw_serde]
+pub struct InstantiateMsg {
+    pub admin: Option<String>,
+    pub max_bundles: Option<u64>,
```

Comment:
would love some explanations of what some of these things mean. the vocab here is pretty impenetrable.

Reply:
got it.